web/public/js/controllers/dojos-map-controller.js

```diff
@@ -43,11 +43,27 @@ function cdDojosMapCtrl($scope, $window, $state, $stateParams, $translate, $geol
     });
 
     if($scope.model.map) {
-      $scope.model.map.setZoom(2);
-      $scope.model.map.setCenter(centerLocation);
       $scope.searchResult = null;
       delete $scope.search.dojo;
     }
+
+    if (navigator.geolocation) {
+      navigator.geolocation.getCurrentPosition(function(position) {
+        var pos = {
+          lat: position.coords.latitude,
+          lng: position.coords.longitude
+        };
+        $scope.model.map.setCenter(pos);
+        $scope.model.map.setZoom(6);
```

Comment:
Reason for 6 was because if there is no dojo in your area, it looks like a crapilly empty map @DanielBrierton